invenio_communities/assets/semantic-ui/less/invenio_communities/theme.less

```diff
@@ -143,14 +139,22 @@
 }
 
 .communities-frontpage {
-
   .community-image {
-    background-color: #E1E2E3;
+    background-color: #e1e2e3;
     // Max dimensions based on default logo
     max-height: 173px;
     max-width: 209px;
     height: 173px;
     overflow: hidden;
   }
+}
 
+.community-records-search {
+  .ui.container {
+    .ui.relaxed.grid {
```

Comment:
I am still not sure about this, sorry :) there is too much parent child dependencies in this rule. Fe. if someone changes the `.relaxed` class, the rule will break immediately. if you have time we could maybe see it together